hack/make/.build-rpm/docker-engine.spec

```diff
@@ -84,9 +84,12 @@ Requires: device-mapper >= 1.02.90-2
 %if 0%{?fedora} >= 22
 %global selinux_policyver 3.13.1-128
 %endif # fedora 22
-%if 0%{?centos} >= 7 || 0%{?rhel} >= 7 || 0%{?oraclelinux} >= 7
+%if 0%{?centos} >= 7 || 0%{?rhel} >= 7
 %global selinux_policyver 3.13.1-23
-%endif # centos,oraclelinux 7
+%endif # centos,rhel 7
+%if 0%{?oraclelinux} >= 7
+%global selinux_policyver 3.13.1-102.0.3.el7_3.15
```

Comment:
Thanks for your suggestion. It is ok to make a special-case OL7 again, but really maybe I fail to see any benefit for that, here is my reason, in this PR, we make a `docker-engine` dependency on `selinux-policy >= 3.13-102.0.3.el7_3.15` in `hack/make/.build-rpm/docker-engine.spec`, so if the `docker-engine` RPM is built with this, it will have the following 2 selinux dependencies:
```
# rpm -qR docker-engine | grep selinux
docker-engine-selinux >= 1.12.6-1.0.3.el7
selinux-policy >= 3.13.1-102.0.3.el7_3.15

```
So if the above `docker-engine` RPM is installed, it should already pulled 2 dependencies with it, i.e `docker-engine-selinux` and `selinux-policy` of at least the specified version. If the system also has `selinux-policy-targeted` installed, it will also get updated, since `selinux-policy-targeted` has dependency on `selinux-policy`. 

I can see the following scenario where there is some need to use a **greater than** version of `docker-engine-selinux` policy with **lower** version `docker-engine` then, the `selinux-policy-targeted` will not get updated appropriately base on the current spec file, but how valid is this scenario ? Even if that case exists, wouldn't it make more sense to also upgrade the `docker-engine` too ?

Not sure if this is all make sense, but that is my understanding of it, maybe I am wrong?